plugins/xcmetrics/src/components/BuildList/BuildList.test.tsx

```diff
@@ -53,7 +53,7 @@ describe('BuildList', () => {
     );
 
     userEvent.click(
-      (await rendered.findAllByLabelText('Detail panel visiblity toggle'))[0],
+      (await rendered.findAllByLabelText('Detail panel visibility toggle'))[0],
```

Comment:
Turns out this spelling mistake comes from `material-ui/table` :joy: so we have to spell this wrong :(